### PR TITLE
Adding readme in cnbinfo

### PR DESCRIPTION
**Pull request by jorgebalsa**
This is just a test. I added a readme.rst in cnbinfo.
There are also other test files.

@adrianq 

[Trello card url](https://trello.com/c/W2V98eAY/160-review-user-machines)

### CODE REVIEW

**Comment by jorgebalsa** [comment]
@adrianq 
(Files removed)
Ready to test

File: cnbinfo/readme.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Description
```
Comment (adrianq) [inline cnbinfo/readme.rst]: @jorgebalsa Have you had a look at how the README.rst looks like? If you click on 'View file', you can have a look at it and the layout doesn't look particularly good. Can you please improve it a bit by highlighting titles, define the code as code, etc?

File: cnbinfo/readme.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Is a python script that get all the usernames and ids for a given users list.
+First, we need to generate this user list file. You can generate it by running:
+  ./multirun.py --literal 'cat /etc/passwd' > users.txt
```
Comment (adrianq) [inline cnbinfo/readme.rst]: Can we define these lines as code to improve readability?

Comment (adrianq) [reply cnbinfo/readme.rst]: Can you please specify where to find multirun script?

File: cnbinfo/readme.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+And then you can run  ./extract_users.py without arguments.
```
Comment (adrianq) [inline cnbinfo/readme.rst]: For this to work you need to copy users.txt to this folder, right?

File: cnbinfo/readme.rst (hunk 2 above)
Comment (adrianq) [reply cnbinfo/readme.rst]: Can you show an example of how the users.txt file should look like?

File: cnbinfo/readme.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Usage
```
Comment (adrianq) [inline cnbinfo/readme.rst]: This is redundant we can probably have one single section, don't you think?

File: cnbinfo/readme.rst (hunk 1 above)
Comment (jorgebalsa) [reply cnbinfo/readme.rst]: @adrianq 
I think I need to add a text editor to github. Do I? Which one do you recomend?
The text editor I have don't let me improve any style.

File: cnbinfo/readme.rst (hunk 2 above)
Comment (jorgebalsa) [reply cnbinfo/readme.rst]: Done

File: cnbinfo/readme.rst (hunk 3 above)
Comment (jorgebalsa) [reply cnbinfo/readme.rst]: Yes

File: cnbinfo/readme.rst (hunk 4 above)
Comment (jorgebalsa) [reply cnbinfo/readme.rst]: Ok. I write Usage now as a bold word

**Comment by adrianq** [comment]
I have just implemented some minor changes. Merging this PR now

**Pull request merged**